docs/reference/core-concepts.md

```diff
@@ -7,3 +7,73 @@ title: Great Expectations Core Concepts
 # Data Docs
 
 # Profiling
+
+Great Expectations provides a mechanism to automatically generate expectations, using a feature called a **Profiler**. A Profiler builds an Expectation Suite from one or more Data Assets. It may also validates the data against the newly-generated Expectation Suite to return a Validation Result. There are several Profilers included with Great Expectations.
```

Comment:
Should the first instance of each GE-specific term have syntax highlighting/bolding/linking? We highlight `Expectation Suite` in  the `## Rule-based Profilers` section but not here.

If the assumption is that the user is fully knowledgeable about these topics when getting to this tutorial, this is perhaps more of a nitpick than anything concrete.